src/5B320.c

```diff
@@ -3,6 +3,175 @@
 #include "item_entity.h"
 #include "world/world.h"
 
+#if !defined(VERSION_JP)
```

Comment:
Better do to `ifdef VERSION_US` I think?

Reply:
This is about compatibility defines; JP has strings for unused items. US (or CN) doesn't have strings for these. So the defines wouldn't get generated.